### PR TITLE
[IDP] Language selector component

### PR DESCRIPTION
**Pull request by mitchmoccia**
Version one of the language selector component.

### CODE REVIEW

**Comment by thisisdano** [comment]
@mitchmoccia When the languages button has its menu active, selecting the button again should dismiss the menu. As far as I can tell, it does not.

**Comment by mitchmoccia** [comment]
> @mitchmoccia When the languages button has its menu active, selecting the button again should dismiss the menu. As far as I can tell, it does not.

@thisisdano This has been addressed. The language selector drop down now toggles when you select the language button over and over. Tested on all examples (three or more, header example, and unstyled) and also with the keyboard.

File: packages/usa-header/_index.scss [hunk 1]
```diff
@@ -1,8 +1,6 @@
 // dependencies
 @forward "uswds-fonts";
 @forward "uswds-helpers";
-@forward "usa-skipnav";
-@forward "usa-search";
```
[the remark this answers is not in the document]
Comment (mitchmoccia) [reply packages/usa-header/_index.scss:5]: @thisisdano I show those as having been removed 8 months ago with the commit message "Add dependencies to component packages"

image

[the remark this answers is not in the document]
Comment (thisisdano) [reply packages/usa-header/_index.scss:5]: I'm going to restore them

File: packages/usa-language-selector/src/usa-language-selector--unstyled/usa-language-selector--unstyled.twig [hunk 2]
```diff
@@ -0,0 +1,27 @@
+<div class="usa-language-container">
+  <ul class="usa-language__primary usa-accordion">
+    <li class="usa-language__primary-item">
+      <button type="button" class="usa-button--unstyled usa-language__link{{ focus ? " usa-focus" }}" {% if role %} role="{{ role }}" {% endif %} aria-expanded="false" aria-controls="language-options">
```
[the remark this answers is not in the document]
Comment (mitchmoccia) [reply packages/usa-language-selector/src/usa-language-selector--unstyled/usa-language-selector--unstyled.twig]: Done! TY

**Review by thisisdano — APPROVED** [review]

**Pull request merged**